src/sentry/deletions/defaults/group.py

```diff
@@ -148,17 +151,15 @@ def delete_bulk(self, instance_list):
         # Remove group objects with children removed.
         return self.delete_instance_bulk(instance_list)
 
-    def delete_instance(self, instance):
+    def delete_instance(self, instance: Group) -> None:
         from sentry import similarity
 
         if not self.skip_models or similarity not in self.skip_models:
             similarity.delete(None, instance)
 
         return super().delete_instance(instance)
 
-    def mark_deletion_in_progress(self, instance_list):
-        from sentry.models.group import Group, GroupStatus
-
+    def mark_deletion_in_progress(self, instance_list: Sequence[Group]) -> None:
```

Comment:
these are likely `Iterable[Group]` instead -- since a `QuerySet[Group]` is not a `Sequence[Group]`